README.md

```diff
@@ -1,14 +1,15 @@
 ![logo](https://raw.github.com/apiaryio/api-blueprint/master/assets/logo_apiblueprint.png)
 
-# Protagonist
+# Protagonist - API Blueprint Parser for Node.js
 
 [![Circle CI](https://circleci.com/gh/apiaryio/protagonist.svg?style=shield)](https://circleci.com/gh/apiaryio/protagonist)
 [![AppVeyor](https://ci.appveyor.com/api/projects/status/uaa6ivk97urmoucr/branch/master?svg=true)](https://ci.appveyor.com/project/Apiary/protagonist)
 
-### API Blueprint Parser for Node.js
-Protagonist is a Node.js wrapper for the [Drafter](https://github.com/apiaryio/drafter) library.
+Protagonist is a Node.js wrapper for the
+[Drafter](https://github.com/apiaryio/drafter) library.
 
-API Blueprint is Web API documentation language. You can find API Blueprint documentation on the [API Blueprint site](https://apiblueprint.org).
+API Blueprint is Web API documentation language. You can find API Blueprint
```

Comment:
I'd rewrite the intro to something like this:

> Protagonist is a Node.js wrapper for the [Drafter] library, an API Blueprint parser. [API Blueprint] is a language for describing web APIs.

Reply:
Agreed